### PR TITLE
[Power Automate] (Limitations) Add parameter size limit

### LINKED ISSUE #554: Max request body size limit (30,000,000 bytes or 28.6MB) when running Office Scripts in Power Automate
When running Office Scripts in Power Automate, the parameters passed into the Run script action can sometimes be very large and its request might get rejected by our service due to the [default ASP.NET Core max request body size limit - 30,000,000 bytes or 28.6MB](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.server.kestrel.core.kestrelserverlimits.maxrequestbodysize?view=aspnetcore-6.0#microsoft-aspnetcore-server-kestrel-core-kestrelserverlimits-maxrequestbodysize).

We have seen customer whose flow passes in a giant JSON (>40MB) as the script parameter and the request failed right away.

Should we callout this platform limit in the documentation?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4085c6ff-856c-4704-8d72-5ef0ca1f64c1
* Version Independent ID: 92eda514-0e20-3add-4b61-62d82d210549
* Content: [Platform limits and requirements with Office Scripts - Office Scripts](https://learn.microsoft.com/en-us/office/dev/scripts/testing/platform-limits)
* Content Source: [docs/testing/platform-limits.md](https://github.com/OfficeDev/office-scripts-docs/blob/main/docs/testing/platform-limits.md)
* Product: **excel**
* Technology: **scripts**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

### PR DESCRIPTION
**Pull request by AlexJerabek**
Fixes #554.

### CODE REVIEW

File: docs/testing/platform-limits.md [hunk 1]
```diff
@@ -26,16 +26,17 @@ Excel limits that aren't specific to Office Scripts can be found in the article
 
 ### Power Automate
 
-When using Office Scripts with Power Automate, each user is limited to **1,600 calls to the Run Script action per day**. This limit resets at 12:00 AM UTC.
+The following limitations with the Power Automate platform are the ones you'll most likely encounter.
 
-The Power Automate platform also has usage limitations, which can be found in the following articles.
+- Each user is limited to **1,600 calls** to the Run Script action per day. This limit resets at 12:00 AM UTC.
+- There is a **120-second timeout** for [synchronous Power Automate operations](/power-automate/limits-and-config#timeout). For long-running scripts, you must either [optimize your script](../develop/web-client-performance.md) or split your Excel automation into multiple scripts.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/platform-limits.md:32]:
```suggestion
- There's a *120-second timeout* for [synchronous Power Automate operations](/power-automate/limits-and-config#timeout). For long-running scripts, you must either [optimize your script](../develop/web-client-performance.md) or split your Excel automation into multiple scripts.
```

File: docs/testing/platform-limits.md [hunk 2]
```diff
@@ -26,16 +26,17 @@ Excel limits that aren't specific to Office Scripts can be found in the article
 
 ### Power Automate
 
-When using Office Scripts with Power Automate, each user is limited to **1,600 calls to the Run Script action per day**. This limit resets at 12:00 AM UTC.
+The following limitations with the Power Automate platform are the ones you'll most likely encounter.
 
-The Power Automate platform also has usage limitations, which can be found in the following articles.
+- Each user is limited to **1,600 calls** to the Run Script action per day. This limit resets at 12:00 AM UTC.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/platform-limits.md:31]:
```suggestion
- Each user is limited to *1,600 calls* to the **Run script** action per day. This limit resets at 12:00 AM UTC.
```

File: docs/testing/platform-limits.md [hunk 3]
```diff
@@ -26,16 +26,17 @@ Excel limits that aren't specific to Office Scripts can be found in the article
...
+- There is a **120-second timeout** for [synchronous Power Automate operations](/power-automate/limits-and-config#timeout). For long-running scripts, you must either [optimize your script](../develop/web-client-performance.md) or split your Excel automation into multiple scripts.
+- The maximum size of parameters passed to the Run script action is **30,000,000 bytes (28.6MB)**.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/platform-limits.md:33]:
```suggestion
- The maximum size of parameters passed to the **Run script** action is *30,000,000 bytes (28.6MB)*.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
I think we should bold the "Run script" text for the action so consider italicizing the limits and compare with how it looks to have both pieces of data bolded.

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT,
Hmm... I think your suggestion is correct by the style guidelines and for consistency with how we refer to **Run script**. But I also think the italics get lost. Usually, when we refer to buttons and proper names, we're getting people to click on them. Here, that's secondary information.

image

File: docs/testing/platform-limits.md (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/testing/platform-limits.md:31]:
```suggestion
- Each user is limited to **1,600 calls** to the Run script action per day. This limit resets at 12:00 AM UTC.
```

File: docs/testing/platform-limits.md (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/testing/platform-limits.md:32]:
```suggestion
- There's a **120-second timeout** for [synchronous Power Automate operations](/power-automate/limits-and-config#timeout). For long-running scripts, you must either [optimize your script](../develop/web-client-performance.md) or split your Excel automation into multiple scripts.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by ElizabethSamuel-MSFT** [comment]
> @ElizabethSamuel-MSFT, Hmm... I think your suggestion is correct by the style guidelines and for consistency with how we refer to **Run script**. But I also think the italics get lost. Usually, when we refer to buttons and proper names, we're getting people to click on them. Here, that's secondary information.
> 
> image

@AlexJerabek I see what you mean. I've updated my feedback.

**Pull request merged**